### PR TITLE
Launcher for quick tests

### PR DESCRIPTION
**Pull request by yschimke**
Use junit5 tests to tag slow tests and run everything but across the project 
image

### CODE REVIEW

**Review by swankjesse — COMMENTED** [review]
What qualifies a test as fast or slow? Excluding CallTest surprises me; it's big but is it also slow?

Maybe I'd argue we should constrain this to tests that sleep?

**Comment by yschimke** [comment]
image

I'll move from the top of CallTest to individual slow tests, at least one takes 10 seconds.

**Comment by yschimke** [comment]
> What qualifies a test as fast or slow? Excluding CallTest surprises me; it's big but is it also slow?
> 
> Maybe I'd argue we should constrain this to tests that sleep?

Ignoring the why it's slow aspect, I found myself not running tests locally because they take minutes.  Including most of CallTest, and it taking 12 seconds seems like a reasonable compromise.

File: okhttp/src/test/java/okhttp3/KotlinSourceModernTest.kt [hunk 1]
```diff
@@ -95,6 +97,11 @@ import org.junit.jupiter.api.Test
 )
 @Disabled
 class KotlinSourceModernTest {
+  @BeforeEach
+  fun disabled() {
```
Comment (swankjesse) [inline okhttp/src/test/java/okhttp3/KotlinSourceModernTest.kt:101]: What's this for?

**Review by swankjesse — APPROVED** [review]
Any guidelines for authors of new tests? What's the cutoff?

**Comment by yschimke** [comment]
> 
> 
> Any guidelines for authors of new tests? What's the cutoff?

2 second is Slow
> 0.5 is Slowish

But overall goal is that probably more like, "across whole project"

!Slow -> 45 seconds
!Slow & ! Slowish -> 10 seconds

File: okhttp/src/test/java/okhttp3/KotlinSourceModernTest.kt (hunk 1 above)
Comment (yschimke) [reply okhttp/src/test/java/okhttp3/KotlinSourceModernTest.kt:101]: @Disabled wasn't working in this setup, it was running it anyway.  And timing out and spewing errors from TODO().  There might be some correct fix, but like the Android 5 junit runner which doesn't support automatic registration of extensions, each new integration has a non zero chance of not handling some feature of test running.

**Pull request merged**